src/www/ui_components/add-server-view.js

```diff
@@ -165,7 +165,7 @@ Polymer({
       <div class="footer center top-divider">
         <div
           id="addServerFooter"
-          inner-h-t-m-l="[[localize('server-create-your-own', 'breakLine', '<br/>', 'openLink', '<a href=https://s3.amazonaws.com/outline-vpn/index.html>', 'closeLink', '</a>')]]"
+          inner-h-t-m-l="[[localize('server-create-your-own', 'breakLine', '<br/>', 'openLink', '<a href=https://s3.amazonaws.com/outline-vpn/index.html>', 'openLink2', '<a href=https://www.reddit.com/r/outlinevpn/wiki/index/outline_vpn_access_keys/>', 'closeLink', '</a>')]]"
```

Comment:
I was thinking maybe we don't want to hard-code the url but... the other one is already here, so...